tools/gfx_tools/compose.py

```diff
@@ -528,21 +528,17 @@ def walk_dirs(self) -> None:
         '''
         Find and process all JSON and PNG files within sheet directory
         '''
-        all_files = sorted(os.walk(self.subdir_path), key=lambda d: d[0])
-        excluded_paths = [  # TODO: dict by parent dirs
-            self.subdir_path.joinpath(ignored_path)
```

Comment:
Needs to be revert this!

(only specific directories should be excluded not a name pattern through the tree)

Reply:
This is outdated, even if it hasn't been flagged.